_posts/0000-01-02-Endonerios.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
```

Comment:
plz plzplzpzlzpzlzpl where is my aghanim wtf noobs?????????????????????????????????????????????????????????????????????//

Reply:
Dank meme refers to viral internet content that, due to overuse or passing trends, has lost its value or currency. It can also refer to exceptionally unique or odd memes.